globals.d.ts

```diff
@@ -365,8 +365,9 @@ declare namespace Spicetify {
         /**
          * Create a single toggle.
          */
+        type Icon = "album" | "artist" | "block" | "brightness" | "car" | "chart-down" | "chart-up" | "check" | "check-alt-fill" | "chevron-left" | "chevron-right" | "chromecast-disconnected" | "clock" | "collaborative" | "computer" | "copy" | "download" | "downloaded" | "edit" | "enhance" | "exclamation-circle" | "external-link" | "facebook" | "follow" | "fullscreen" | "gamepad" | "grid-view" | "heart" | "heart-active" | "instagram" | "laptop" | "library" | "list-view" | "location" | "locked" | "locked-active" | "lyrics" | "menu" | "minimize" | "minus" | "more" | "new-spotify-connect" | "offline" | "pause" | "phone" | "play" | "playlist" | "playlist-folder" | "plus-alt" | "plus2px" | "podcasts" | "projector" | "queue" | "repeat" | "repeat-once" | "search" | "search-active" | "shuffle" | "skip-back" | "skip-back15" | "skip-forward" | "skip-forward15" | "soundbetter" | "speaker" | "spotify" | "subtitles" | "tablet" | "ticket" | "twitter" | "visualizer" | "voice" | "volume" | "volume-off" | "volume-one-wave" | "volume-two-wave" | "watch" | "x";
         class Item {
-            constructor(name: string, isEnabled: boolean, onClick: (self: Item) => void);
+            constructor(name: string, isEnabled: boolean, onClick: (self: Item) => void, icon: Icon | string);
```

Comment:
Passing in an icon should not be mandatory, nullish coalescing is already present in the wrapper.
```suggestion
            constructor(name: string, isEnabled: boolean, onClick: (self: Item) => void, icon?: Icon | string);
```

Reply:
sry I dont know much about `*.d.ts` files